infrastructure/modules/dynamo_db/main.tf

```diff
@@ -63,6 +63,7 @@ resource "aws_iam_policy" "dynamodb_policy" {
         "Effect" : "Allow",
         "Action" : [
           "dynamodb:Query",
+          "dynamodb:Scan",
```

Comment:
Should we add this in the module if it's just being used for the back channel logout? maybe it would be better to have a small policy resource instead?

Reply:
Rich and I decided to add it as default as we should be able to scan a dynamo db from any lambda.